tests/test-is-cookie-domain-acceptable.c

```diff
@@ -109,6 +113,18 @@ static void test_psl(void)
 
 int main(int argc, const char * const *argv)
 {
+#ifdef _WIN32
+	WSADATA wsa_data;
+	int err;
+
+	if ((err = WSAStartup(MAKEWORD(2,2), &wsa_data))) {
+		printf("WSAStartup failed with error: %d\n", err);
+		return 1;
+	}
+
+	atexit((void (__cdecl*)()) WSACleanup);
```

Comment:
My bad since MSVC warns:
```
warning C4113: 'void (__cdecl *)()' differs in parameter lists from 'void (__cdecl *)(void)'
```
Casting like `atexit ((void (__cdecl*)(void)) WSACleanup);` gives no warning.

Reply:
Thanks, force pushed the change.